src/Symfony/Component/Form/Event/PostSubmitEvent.php

```diff
@@ -21,4 +21,11 @@
  */
 final class PostSubmitEvent extends FormEvent
 {
+    /**
+     * @deprecated since Symfony 6.4, it will throw an exception in 7.0.
+     */
+    public function setData(mixed $data)
```

Comment:
```suggestion
    public function setData(mixed $data): void
```

Reply:
Done, I did not notice these were final 🙈, thanks!
Should it be changed to `never` in 7.0?